controllers/dockermachine_controller.go

```diff
@@ -115,6 +115,12 @@ func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		return ctrl.Result{}, err
 	}
 
+	//delete dockerMachine
+	if !machine.ObjectMeta.DeletionTimestamp.IsZero() {
+		log.Info("Start delete dockerMachine")
+		return r.delete(ctx, cluster, machine, dockerMachine)
+	}
+
 	// creating loadbalancer
 	log.Info("Reconciling cluster")
```

Comment:
@chuckha Not entirely related to this PR, but shouldn't this be "Reconciling DockerMachine" rather than cluster?

Reply:
yeah, we've officially overloaded the term cluster and machine :D